common/src/main/java/org/apache/atlas/repository/Constants.java

```diff
@@ -324,6 +328,20 @@ public enum SupportedFileExtensions { XLSX, XLS, CSV }
 
     public static final int ELASTICSEARCH_PAGINATION_SIZE = 50;
 
+    public static final String CATALOG_PROCESS_INPUT_RELATIONSHIP_LABEL = "__Process.inputs";
+    public static final String CATALOG_PROCESS_OUTPUT_RELATIONSHIP_LABEL = "__Process.outputs";
+    public static final String COLUMN_LINEAGE_RELATIONSHIP_LABEL = "__Process.columnProcesses";
+    public static final String CLASSIFICATION_PROPAGATION_MODE_DEFAULT  ="DEFAULT";
+    public static final String CLASSIFICATION_PROPAGATION_MODE_LINEAGE  ="LINEAGE";
+
+    public static final HashMap<String, ArrayList<String>> CLASSIFICATION_PROPAGATION_EXCLUSION_MAP = new HashMap<String, ArrayList<String>>(){{
+        put(CLASSIFICATION_PROPAGATION_MODE_DEFAULT, new ArrayList<>(Arrays.asList(CATALOG_PROCESS_INPUT_RELATIONSHIP_LABEL,
```

Comment:
These 3 labels will be part of this object CLASSIFICATION_PROPAGATION_MODE_RESTRICT_LINEAGE

Reply:
Yeah that was mistake